@here/harp-mapview/lib/MapView.ts

```diff
@@ -602,6 +603,11 @@ export interface MapViewOptions {
      * @default false.
      */
     synchronousRendering?: boolean;
+
+    /**
+     * URL to watermark logo to be displayed in bottom left corner.
+     */
+    watermark?: string;
```

Comment:
Pass it through.
```
WatermarkOptions {
    imageUrl: ..
    width: ... // defaults to texture image width
    height: ... // defaults to texture image width
    margin ...
}
```
And make `Watermark` object accept more complex options.

Or just allow passing ready `Watermark` object.